packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/queue/queue.ts

```diff
@@ -0,0 +1,389 @@
+import * as path from 'path';
+import * as cloudwatch from '@aws-cdk/aws-cloudwatch';
```

Comment:
Is there a diff of what changed between the old `queue.ts` and this one? I just want to make sure that I am reviewing the right stuff